static/src/js/components/GridRow/GridRow.jsx

```diff
@@ -34,6 +35,8 @@ const GridRow = (
     registry,
     required,
     schema,
+    idSchema,
```

Comment:
This idSchema is needed for GridGroupedSinglePanel and GridCheckboxPanel because both of these components are calling GridLayout recursively, so in order to for GridLayout to have access to idSchema I am passing it down through GridRow.